examples/monitor_list.rs

```diff
@@ -1,13 +1,38 @@
 #![allow(clippy::single_match)]
 
 use simple_logger::SimpleLogger;
+use winit::monitor::MonitorHandle;
 use winit::{event_loop::EventLoop, window::WindowBuilder};
 
 fn main() {
     SimpleLogger::new().init().unwrap();
     let event_loop = EventLoop::new();
     let window = WindowBuilder::new().build(&event_loop).unwrap();
 
-    dbg!(window.available_monitors().collect::<Vec<_>>());
-    dbg!(window.primary_monitor());
+    for mon in window.available_monitors() {
+        print_info(mon);
+    }
+    if let Some(mon) = window.primary_monitor() {
```

Comment:
It is not very clear that the last monitor is the primary monitor (if there is one...).
I suggest to highlight the primary monitor when looping over all the monitors.